snuba/query/processors/logical/filter_in_select_optimizer.py

```diff
@@ -43,36 +45,57 @@ class FilterInSelectOptimizer:
     def process_mql_query(
         self, query: LogicalQuery | CompositeQuery[QueryEntity]
     ) -> None:
-        feat_flag = get_int_config("enable_filter_in_select_optimizer", default=0)
+        feat_flag = get_int_config("enable_filter_in_select_optimizer", default=1)
         if feat_flag:
             try:
                 domain = self.get_domain_of_mql_query(query)
-            except ValueError:
+            except ValueError as err:
+                sentry_sdk.capture_exception(err)
```

Comment:
I am worried that this is going to trigger our Sentry alerts. We discussed maybe using `logger.warning` as an alternative, so these get sent to Sentry as warnings instead of full errors.

Reply:
oh yes forgot about this, will change